modules/uf2/CMakeLists.txt

```
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 Pete Johanson
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_BUILD_OUTPUT_BIN AND CONFIG_BUILD_OUTPUT_UF2)
+  set_property(GLOBAL APPEND PROPERTY
+    extra_post_build_commands
+    COMMAND ${PYTHON_EXECUTABLE} ${ZEPHYR_BASE}/../tools/uf2/utils/uf2conv.py
```

When we have this as a Zephyr module, you should use:
```suggestion
    COMMAND ${PYTHON_EXECUTABLE} ${ZEPHYR_UF2_MODULE_DIR}/utils/uf2conv.py
```